DOCS/waf-buildsystem.rst

```diff
@@ -68,7 +68,7 @@ consists of running ``pkg-config`` and looking for ``vdpau`` with version
 ``>= 0.2``. If the check succeeds a ``#define HAVE_VDPAU 1`` will be added to
 ``config.h``, if not ``#define HAVE_VDPAU 0`` will be added.
 
-The defines names are automatically prepended with ``HAVE_``, capitalized and
+The defined names are automatically prepended with ``HAVE_``, capitalized and
```

Comment:
the define here refers to the #define directive. so shouldn't it be just "The define names" or "The #define names", but definitely not defined i think.

Reply:
Ok, but wouldn't it make more sense to use ``define`` in this context?

Reply:
if you meant what i first suggested just with formatting, sure.